docker/docker-compose.yml

```diff
@@ -319,3 +319,12 @@ services:
     # See the following:
     # - https://www.elastic.co/guide/en/elastic-stack-get-started/current/get-started-docker.html,
     # - https://github.com/deviantony/docker-elk/issues/243
+
+  rss-bridge:
+    image: rssbridge/rss-bridge:2022-01-20
+    container_name: 'rss-bridge'
+    volumes:
+      - ../config/rss-bridge:/config
+    ports:
+      - 3123:80
```

Comment:
We can't bind this to port 80 on the host, as this is occupied by Traefik (our API router).  Traefik is responsible for routing all requests to our services via a unified URL structure (e.g., Posts service is at `/v1/posts`, the Search service is at `/v1/search`, etc.)

This new RSS-Bridge service can be made to follow the same pattern, if we add Traefik routing information to the container.  We do that with custom labels.  In `docker/docker-compose.yml` do this:

```yml
  rss-bridge:
    image: rssbridge/rss-bridge:2022-01-20
    container_name: 'rss-bridge'
    volumes:
      - ../config/rss-bridge:/config
    depends_on:
      - traefik
    labels:
      # Enable Traefik
      - 'traefik.enable=true'
      # Traefik routing for the rss-bridge service at /v1/rss-bridge
      - 'traefik.http.routers.search.rule=PathPrefix(`/${API_VERSION}/rss-bridge`)'
      # Specify the RSS-Bridge service port
      - 'traefik.http.services.search.loadbalancer.server.port=3123'
      # Add middleware to this route to strip the /v1/rss-bridge prefix
      - 'traefik.http.middlewares.strip_rss_bridge_prefix.stripprefix.prefixes=/${API_VERSION}/rss-bridge'
      - 'traefik.http.middlewares.strip_rss_bridge_prefix.stripprefix.forceSlash=true'
      - 'traefik.http.routers.rss_bridge.middlewares=strip_search_prefix'
```

Then in `docker/production.yml` add this (to have the container auto-restart if it crashes):

```yml
rss-bridge:
  restart: unless-stopped
```

Now when you start the services (`pnpm services:start`) you should have http://localhost/v1/rss-bridge available, and similarly on api.telescope.cdot.systems/v1/rss-bridge in production and dev.api.telescope.cdot.systems/v1/rss-bridge in staging.

Reply:
I have one small problem.

So, I was testing whether the service is working fine, but it seems that it is lacking some static files. Instead of hitting `localhost/v1/rss-bridge/static`, the requests hit at `localhost/v1/static`. Is there a way to change that with `traefik`?

Reply:
Can you push up the code you have so I can see it?  I notice I have a typo above:

```
- 'traefik.http.routers.rss_bridge.middlewares=strip_search_prefix'
```

should be:

```
- 'traefik.http.routers.rss_bridge.middlewares=strip_rss_bridge_prefix'
```

Reply:
Sure, I will push what I have